pkg/version/version.go

```diff
@@ -328,12 +328,12 @@ func IsSupported(haveVersion, wantVersion string) (bool, error) {
 
 	supportedVersion, err := semver.Parse(wantVersion)
 	if err != nil {
-		return false, err
+		return false, errors.WithStack(err)
 	}
 
 	currentVersion, err := semver.Parse(haveVersion)
 	if err != nil {
-		return false, err
+		return false, errors.WithStack(err)
```

Comment:
Is the stack useful for the end-user?

Reply:
I noticed we use it in other version check funcs such as `VersionIsEqualTo(...` in this file. It can be useful to the user with `--stack-trace` flag